app/models/organization_campaign.rb

```diff
@@ -35,7 +35,7 @@ def matched
 
   def matched_pct
```

Comment:
A [guard clause](http://www.thechrisoshow.com/2009/02/16/using-guard-clauses-in-your-ruby-code/) here might be more clear:

```
def matched_pct # would do matched_percent, just because I rarely abbreviate
  return 0 if assigned == 0

  ((matched.to_f / assigned.to_f) * 100.00).to_i
end
```